commcare_connect/opportunity/forms.py

```diff
@@ -20,9 +20,12 @@ def __init__(self, *args, **kwargs):
             Row(Field("name")),
             Row(Field("description")),
             Row(Field("active")),
+            Row(Field("users")),
             Submit("submit", "Submit"),
         )
 
+        self.fields["users"] = forms.FileField()
```

Comment:
What kind of file are you expecting here? I imagine that the most likely type would be an Excel file.

If we want to support a plain text file we could just have a text box that you paste the numbers into.

Reply:
> If we want to support a plain text file we could just have a text box that you paste the numbers into.

That seems like a nice idea for now, so I switched to this.

Reply:
Did you mean you switched to a text box? I still see a `FileField`.

Reply:
Yep, I apparently stashed that change instead of committing it